web/scm/src/app/side-panel/side-panel.component.ts

```diff
@@ -1,15 +1,95 @@
 import { Component, OnInit } from '@angular/core';
+import { FormGroup, FormControl } from '@angular/forms';
 
 @Component({
   selector: 'scm-side-panel',
   templateUrl: './side-panel.component.html',
   styleUrls: ['./side-panel.component.scss']
 })
 export class SidePanelComponent implements OnInit {
+  form: FormGroup;
 
-  constructor() { }
+  productSelectOptions: any[];
```

Comment:
Can it be object or Map of strings? I think any is too wide

Reply:
Got it. It is too wide, but I thought since I just made up dummy data, I will leave it wide and come back and change it when we have the actual format. But I might forget, so it is better I narrow it now and change it later.